ui/main.slint

```diff
@@ -0,0 +1,106 @@
+import { VerticalBox,  ScrollView, StandardButton, Palette } from "std-widgets.slint";
+import { Blur, LineEditInfo, TopPanel } from "utils.slint";
+import { BottomButton } from "button.slint";
+import { MenuScreen } from "screens/menu.slint";
+import { AddProfileScreen } from "screens/add_profile.slint";
+import { ProfilesScreen } from "screens/profiles.slint";
+import { MessageScreen } from "screens/message.slint";
+
+export component AppWindow inherits Window {
+    property <[string]> button_str: ["Connect", "Disconnect", "Disconnecting...", "Connecting..."];
+    property <[string]> status_str: ["Disconnected", "Connected", "Disconnecting", "Connecting"];
+
+    in property <string> window_title;
+    in property <int> status;
+    in property <bool> freeze;
+    in property <[LineEditInfo]> profiles <=> profiles_screen.info;
+    in-out property <string> message;
+    in-out property <[LineEditInfo]> manual: [
+        { label: "Address:", text: "" },
+        { label: "Port:", text: "" },
+        { label: "Root certificate:", text: "" },
+        { label: "Certificate:", text: "" },
+        { label: "Key:", text: "" },
+    ];
+    in-out property <LineEditInfo> from_file: { label: "", text: "" };
+    out property <int> menu_index: 0;
+
+    callback browse <=> add_profile_screen.browse;
+    callback reload;
+    callback disconnect();
+    callback connect();
+
+    preferred-width: 1280px;
+    preferred-height: 720px;
+    min-width: 640px;
+    min-height: 360px;
+    title: window_title;
+
+    blur := Blur {
+        z: 2;
+        trigger: parent.freeze;
+        freeze: true;
+    }
+
+    message_screen := MessageScreen {
+        z: 10;
+        text <=> parent.message;
+    }
+
+    top_panel := TopPanel {
+        width: parent.width;
+        text-left: menu_screen.section;
+        text-right: parent.status_str[status] + (parent.status == 1 ? " to " + parent.from_file.label : "");
```

Comment:
Is it possible to replace `1` with named constant, e.g. `STATUS_CODE_CONNECTED`?